doc/devdocs/modules/launcher/how-to-make-a-community-plugin.md

```diff
@@ -0,0 +1,58 @@
+# how-to-make-a-community-plugin
+
+- [ ] Community plugin project name pattern: `Community.PowerToys.Run.Plugin.{PluginName}`
```

Comment:
according to @crutkas' comment this isn't required

Reply:
Namespace

Reply:
shouldn't we differentiate between built-in, community and third-party here to make things clear

Reply:
I don’t want to block this PR on that. I think a quick phrase adjustment would be fine.

Reply:
> I don’t want to block this PR on that. I think a quick phrase adjustment would be fine. 

That's what I mean. No big things.